docs/errors.md

```diff
@@ -62,6 +62,7 @@ SystemJS uses `fetch()` instead of `<script>` to load modules whenever the [shou
 
 SystemJS checks the HTTP [Response object](https://developer.mozilla.org/en-US/docs/Web/API/Response) to check the content-type header. The header must be one of the following:
 
+- `text/javascript`
```

Comment:
While there are a number of JS MIME types, `application/javascript` is the most strongly recommended canonical one.

`text/javascript` is not really encouraged, so it could be nice to reflect this somehow like a section at the end along the lines of:

"Other accepted JS MIME types include `text/javascript`" or something like that to ensure there is a distinction.

Reply:
👍 I created #2179 to change this